packages/flutter_tools/lib/src/commands/devices.dart

```diff
@@ -12,6 +13,14 @@ import '../globals.dart' as globals;
 import '../runner/flutter_command.dart';
 
 class DevicesCommand extends FlutterCommand {
+
+  DevicesCommand() {
```

Comment:
This command also needs a test, see the commands.shard for some examples

Reply:
@jonahwilliams I'm confused. I didn't add this command, I just implemented the constructor (the command did have a default constructor before). And I have tests for the only thing I've added: `--machine` flag. 

Also, the default functionality of the `devices` command does have tests in `test/commands.shard/hermetic/devices_test.dart`

Reply:
(Sorry for the delay)

Right, but you need a test that invokes the --machine and then verifies the output as expected. There are some example tests in tests/commands.shard/permeable/**